dsgrid/registry/registry_manager.py

```diff
@@ -514,3 +527,33 @@ def get_registry_path(registry_path=None):
 def get_registry_class(registry_type):
     """Return the subtype of RegistryBase correlated with registry_type."""
     return _REGISTRY_TYPE_TO_CLASS[registry_type]
+
+
+def _should_use_remote_data():
+    if sys.platform in ("darwin", "win32"):
+        # Local systems need to sync all load data files.
+        use_remote_data = False
+    elif "NREL_CLUSTER" in os.environ:
+        # All Eagle compute nodes have access to the shared load data files.
+        logger.info("Use remote data on NREL_CLUSTER %s", os.environ["NREL_CLUSTER"])
```

Comment:
I don't recall this environment variable. Where should it be documented?

Reply:
Maybe this is supplied by Jade but this is how we determine that the env is on Eagle.

Reply:
In common.py, we used `if os.environ.get("NREL_CLUSTER") is not None:`, is the elif statement equivalent?

Reply:
This is set by the NREL HPC software. It's documented on their website.

Reply:
> In common.py, we used `if os.environ.get("NREL_CLUSTER") is not None:`, is the elif statement equivalent?

Yes, equivalent